ci-operator/step-registry/distributed-tracing/tests/opentelemetry/stage/distributed-tracing-tests-opentelemetry-stage-commands.sh

```
@@ -77,13 +77,13 @@ tests/e2e-prometheuscr \
 tests/e2e-sidecar || any_errors=true
 
 # Set the operator environment variables for metadata filters tests.
-OTEL_CSV_NAME=$(oc get csv -n openshift-opentelemetry-operator | grep "opentelemetry-operator" | awk '{print $1}')
-oc -n openshift-opentelemetry-operator patch csv $OTEL_CSV_NAME --type=json -p '[
+OTEL_CSV_NAME=$(oc get csv -n opentelemetry-operator-system | grep "opentelemetry-operator" | awk '{print $1}')
+oc -n opentelemetry-operator-system patch csv $OTEL_CSV_NAME --type=json -p '[
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)